docs/content.zh/release-notes/flink-1.14.md

```diff
@@ -0,0 +1,424 @@
+situation described by FLINK-21928.
+
+#### Zookeeper node under leader and leaderlatch is not deleted after job finished
+
+##### [FLINK-20695](https://issues.apache.org/jira/browse/FLINK-20695)
+
+The `HighAvailabilityServices` have received a new method `cleanupJobData` which can be implemented
+in order to clean up job related HA data after a given job has terminated.
+
+### Checkpoints
+
+#### Change local alignment timeout back to the global time out
+
+##### [FLINK-23041](https://issues.apache.org/jira/browse/FLINK-23041)
+
+The semantic of alignmentTimeout configuration has changed to such meaning:
+
+The time between the start of a checkpoint(on the checkpoint coordinator) and the time when the
+checkpoint barrier is received by a task.
+
+#### Disable unaligned checkpoints for broadcast partitioning
+
+##### [FLINK-22815](https://issues.apache.org/jira/browse/FLINK-22815)
+
+Unaligned checkpoints were disabled for BROADCAST exchanges.
+
+Broadcast partitioning can not work with unaligned checkpointing. There are no guarantees that
+records are consumed at the same rate in all channels. This can result in some tasks applying state
+changes corresponding to a certain broadcasted event while others don't. In turn upon restore, it
+may lead to an inconsistent state.
+
+#### DefaultCompletedCheckpointStore drops unrecoverable checkpoints silently
+
+##### [FLINK-22502](https://issues.apache.org/jira/browse/FLINK-22502)
+
+On recovery, if a failure occurs during retrieval of a checkpoint, the job is restarted (instead of
+skipping the checkpoint in some circumstances). This prevents potential consistency violations.
+
+#### Recover checkpoints when JobMaster gains leadership
+
+##### [FLINK-22483](https://issues.apache.org/jira/browse/FLINK-22483)
+
+This changes a semi-public interface around CompletedCheckpointStore.
+
+`CompletedCheckpointRecover#recover()` method has been removed as we only need to "recover" once,
+after JobManagerRunner gains leadership. We are now expecting CheckpointRecoveryFactory to always
+return an already recovered CompletedCheckpointStore. To make this new behavior more explicit,
+we've renamed `CheckpointRecoveryFactory#createCompletedCheckpointStore` factory method to
+`CheckpointRecoveryFactory#createRecoveredCompletedCheckpointStore`.
+
+#### Remove CheckpointConfig.setPreferCheckpointForRecovery because it can lead to data loss
+
+##### [FLINK-20427](https://issues.apache.org/jira/browse/FLINK-20427)
+
+Remove deprecated CheckpointConfig#setPreferCheckpointForRecovery
+
+Deprecated method CheckpointConfig#setPreferCheckpointForRecovery was removed, because preferring
+older checkpoints over newer savepoints for recovery can lead to data loss.
+
+### Dependency upgrades
+
+#### Bump up rocksdb version
+
+##### [FLINK-14482](https://issues.apache.org/jira/browse/FLINK-14482)
+
+FRocksDB has been upgraded to 6.20.3-ververica-1.0. Compared to the previous version, the new
```

Comment:
Most users don't know about FRocksDB, that is like an implementation detail. I would talk about this as RocksDB here.

I would also add a slight summary of the performance situation here, the executive summary that all users will be interested in.

Reply:
@Myasuka May I ask you to help me with the summary of the performance implications?

Reply:
How about this:

`
Compared to the previous version, the new version contains lots of bug fix, ARM platform supported, musl library supported, and more attractive features. 
However, the new versin can entail at most 8% performance regression according to our tests (more details can be found in https://issues.apache.org/jira/browse/FLINK-14482).
`